tests/conftest.py

```diff
@@ -168,6 +168,18 @@ def osv_output_missing_cve() -> str:
     return data
 
 
+@pytest.fixture(name="nvd_output")
+def nvd_output() -> str:
```

Comment:
Where is the unit test for the CVE in the reference?

Reply:
It is there